src/common/check_connectivity.cpp

```diff
@@ -60,19 +68,15 @@ Network::Network(void)
         throw std::bad_alloc();
     }
 
-    if ((dnsbase = evdns_base_new(evbase, 0)) == NULL) {
+    if ((dnsbase = evdns_base_new(evbase, 1)) == NULL) {
```

Comment:
According to the libevent docs, the second argument of ```evdns_base_new()``` must be 1 to use the system DNS resolver.